### ISSUE TITLE
Some series content returns empty folders despite it being found in the plugin setup

### ISSUE THREAD

**Issue by ModdingFriendly** [Kevinjil/Jellyfin.Xtream#87]
Sorry my C knowledge is really lacking these. days so I can't quite work out what's happening on my own. 

I just got Xtream working amazingly for most of the content from the server, but when it comes to Series I'm having no luck getting them all to work.

LiveTV & Movies work perfectly but Jellyfin appears to struggle parsing most of the series returning; 

```bash
[15:32:33] [DBG] [16] Jellyfin.Api.ModelBinders.CommaDelimitedArrayModelBinder: Error converting value.
System.FormatException: SongCount is not a valid value for ItemFields.
 ---> System.ArgumentException: Requested value 'SongCount' was not found.
   at System.Enum.TryParseByName(RuntimeType enumType, ReadOnlySpan`1 value, Boolean ignoreCase, Boolean throwOnFailure, UInt64& result)
   at System.Enum.TryParseInt32Enum(RuntimeType enumType, ReadOnlySpan`1 value, Int32 minInclusive, Int32 maxInclusive, Boolean ignoreCase, Boolean throwOnFailure, TypeCode type, Int32& result)
   at System.Enum.TryParse(Type enumType, ReadOnlySpan`1 value, Boolean ignoreCase, Boolean throwOnFailure, Object& result)
   at System.Enum.TryParse(Type enumType, String value, Boolean ignoreCase, Boolean throwOnFailure, Object& result)
   at System.ComponentModel.EnumConverter.ConvertFrom(ITypeDescriptorContext context, CultureInfo culture, Object value)
   --- End of inner exception stack trace ---
   at System.ComponentModel.EnumConverter.ConvertFrom(ITypeDescriptorContext context, CultureInfo culture, Object value)
   at Jellyfin.Api.ModelBinders.CommaDelimitedArrayModelBinder.GetParsedResult(IReadOnlyList`1 values, Type elementType, TypeConverter converter)
```

When trying to enter one of the subfolders of the channel, the rough folder structure appears to be `/series/genre/serial_name/seasons/episodes` 

Any help would be appreciated in case I'm just missing something obvious 🙈

**Comment by ModdingFriendly** [comment]
This feels like it might be linked with #85 as a rough guess as it appears to be a similar problem with Jellyfin loading content within the Series

**Comment by Kevinjil** [comment]
In a few weeks I expect to have some time to pick up the issues.

**Comment by tonyhouse1982** [comment]
Hi @Kevinjil I have the same issue for tv series:
some of them are empty and others are simply missing...
Did You found some time to review this?
I tried to debug by myself but with no luck so far...
I don't understand how jellyfin handle content for those type of sources...
I tried to deselect everything and load only a single serie and the issue is the same...
I don't know when the list is updated (cronjob?)
thanks in advance

**Comment by Z3R093** [comment]
Hi @Kevinjil hope all is well !

Any idea on how i can get the TV series to show from my M3u file. if anyone can provide any information on how to fix this it will be very much appreciated!

**Comment by brianjmurrell** [comment]
I still see this problem -- empty series categories and I am running 0.8.0.0 which surely is much newer than the fix that landed in #97.
